app/components/badge_tooltip_component.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class BadgeTooltipComponent < BaseComponent
+  attr_reader :tag_options, :tooltip_text
+
+  def initialize(tooltip_text:, **tag_options)
+    @tag_options = tag_options
+    @tooltip_text = tooltip_text
+  end
+
+  def call
+    content_tag(
+      :'lg-badge-tooltip',
+      badge_content,
+      'tooltip-text': tooltip_text,
```

Comment:
should we make this a data attribute?
```suggestion
      data: { tooltip_text: tooltip_text },
```

Reply:
Yes that makes sense

Reply:
I haven't seen a strong consensus in the community, but I've personally moved away from `data-` tags for custom elements, and just using the plain unprefixed attribute name.

Reply:
oh I didn't realize you'd moved away, my bad